apis/coralogix/v1alpha1/alert_types.go

```diff
@@ -1053,6 +1061,23 @@ func expandNotification(notification Notification) (*alerts.AlertNotification, e
 	return result, nil
 }
 
+func extractIntegrationID(name string) (uint32, error) {
```

Comment:
Not required, but I'd suggest getIntegration, findIntegration or serachIntegration